src/app/header/header.component.html

```diff
@@ -114,6 +114,9 @@
           </li>
         </ul>
       </li>
+      <li>
+        <a (click)="modal.openAboutModal($event);">About</a>
```

Comment:
This can't go here. I think it should go in the drop down on the right side of the menu. @catrobson can you please confirm.